### DISCUSSION TITLE
Domain configurations jumping between valid and invalid

### DISCUSSION THREAD

**Discussion by jaredrivers** [vercel/vercel#7805] [Help]
After configuring the DNS records and nameservers on my site for the custom domain that I purchased thru domain dot com the list of my domains jump between valid and invalid configuration like every 10 seconds. 

My [site](https://www.jaredriver.com) is currently returning as 'under development' and I have no idea how to fix this.

[Github ](https://github.com/jaredrivers/Portfolio )for the site

**Comment by aufarijaal** [comment]
@jaredrivers same problem happened to me. did you solve the problem?
